### DISCUSSION TITLE
App Crashes

### DISCUSSION THREAD

**Discussion by SilvaLau** [aidanmacgregor/Netflix_ATV_L3_DRM_Uncertified_Mod#23] [General]
I installed this on my H96 MAX 3566, it works great except the occasional crash. I'm not sure what could be causing the issue but every few days after switching on my device, I cannot open the app. When I open it, the screen just turn black for a few moment and then the app crashes. I have to go to app setting to clear the data, then the app works again, though I'd have to re-login account since the data is cleared.

Can anyone help me troubleshoot what could be causing Netflix to crash? Why is it that I have to clear my data once every few days to keep using the app?

**Comment by aidanmacgregor** [comment]
odd i haven't seen this one, seems like its a firmware issue, rhe RK Chips can have very buggy firmware (suppose they all can be even amlogic), not too sure here :)

**Comment by SilvaLau** [comment]
aw... can anything be done? 
Does that mean I'll have to keep resetting my data once every few days or to buy a new tv box? @.@

**Comment by aidanmacgregor** [comment]
not much, apart from try other firmwares for your rockchip device (RK 3566), something else may be more stable :)
